package.json

```diff
@@ -0,0 +1,13 @@
+{
+    "devDependencies": {
+        "prettier": "3.2.5",
+        "prettier-plugin-sh": "0.14.0",
+        "prettier-plugin-toml": "2.0.1"
+    },
+    "prettier": {
```

Comment:
Optional: it might be convenient to add an npm script for either just prettier or prettier + cargo fmt. See e.g. https://github.com/ilyagr/diffedit3/blob/935cbead8fd5ac148a6df44d683fcd327e49b383/package.json#L6.

Npm scripts can be run with `npm run`, which I find slightly nicer than npx. I dislike the latter because it's hard to be sure about whether it's using the global Node environment or the project one, whereas `npm run` will fail outside a project.

Reply:
That's a fair point about `npx`. An `npm` script sounds good, but I think it would be weird if we used it as a general task runner (i.e. if it ran `cargo fmt`).